mlir/lib/IR/AsmPrinter.cpp

```diff
@@ -189,6 +189,11 @@ struct AsmPrinterOptions {
       "mlir-print-value-users", llvm::cl::init(false),
       llvm::cl::desc(
           "Print users of operation results and block arguments as a comment")};
+
+  llvm::cl::opt<bool> printUniqueIDs{
+      "mlir-print-unique-ids", llvm::cl::init(false),
```

Comment:
Should this be `-mlir-print-unique-ssa-ids` to be more explicit? "unique ids" seems a bit generic.

Or an option with a named value: `-mlir-print-ssa-ids-scope={local, global}`

Reply:
Unless you feel strongly it should be a named option, I like `-mlir-print-unique-ssa-ids`.